function.js

```diff
@@ -1,77 +1,43 @@
-}
-
-function myWork() {
-  document.getElementById("home").style.visibility = "hidden";
-  document.getElementById("opening").style.visibility = "hidden";
-  document.getElementById("skillset").style.visibility = "hidden";
-  document.getElementById("work").style.visibility = "visible";
-  document.getElementById("education").style.visibility = "hidden";
-  document.getElementById("interest").style.visibility = "hidden";
-  document.getElementById("reason").style.visibility = "hidden";
-  document.getElementById("misc").style.visibility = "hidden";
-}
-
-function myStudy() {
-  document.getElementById("home").style.visibility = "hidden";
-  document.getElementById("opening").style.visibility = "hidden";
-  document.getElementById("skillset").style.visibility = "hidden";
-  document.getElementById("work").style.visibility = "hidden";
-  document.getElementById("education").style.visibility = "visible";
-  document.getElementById("interest").style.visibility = "hidden";
-  document.getElementById("reason").style.visibility = "hidden";
-  document.getElementById("misc").style.visibility = "hidden";
-}
-
-function myHobby() {
-  document.getElementById("home").style.visibility = "hidden";
-  document.getElementById("opening").style.visibility = "hidden";
-  document.getElementById("skillset").style.visibility = "hidden";
-  document.getElementById("work").style.visibility = "hidden";
-  document.getElementById("education").style.visibility = "hidden";
-  document.getElementById("interest").style.visibility = "visible";
-  document.getElementById("reason").style.visibility = "hidden";
-  document.getElementById("misc").style.visibility = "hidden";
-}
-
-function myReason() {
-  document.getElementById("home").style.visibility = "hidden";
-  document.getElementById("opening").style.visibility = "hidden";
-  document.getElementById("skillset").style.visibility = "hidden";
-  document.getElementById("work").style.visibility = "hidden";
-  document.getElementById("education").style.visibility = "hidden";
-  document.getElementById("interest").style.visibility = "hidden";
-  document.getElementById("reason").style.visibility = "visible";
-  document.getElementById("misc").style.visibility = "hidden";
-}
-
-function myOthers() {
-  document.getElementById("home").style.visibility = "hidden";
-  document.getElementById("opening").style.visibility = "hidden";
-  document.getElementById("skillset").style.visibility = "hidden";
-  document.getElementById("work").style.visibility = "hidden";
-  document.getElementById("education").style.visibility = "hidden";
-  document.getElementById("interest").style.visibility = "hidden";
-  document.getElementById("reason").style.visibility = "hidden";
-  document.getElementById("misc").style.visibility = "visible";
-}
-
+function scrollingTop() {
+  const y = window.scrollY;
+  if(y > 0){
+    window.requestAnimationFrame(scrollingTop);
+    window.scrollTo(0, y - y / 30);
+  }
+}
```

Comment:
There is many ways to do it this method seems a bit hacking. This is just personal preference but you could have done something like this instead as well:
```css
html {
  scroll-behavior: smooth;
}
```
```js
scrollTo(0,0);
```
And achieve the same smooth scrolling effect.

Reply:
Thank you for checking, xuatz. I will rethink on adding and removing codes with comments as well as testing. I also thank Ian for giving the check with help; I'm sorry for letting Ian suggesting the code I was trying to figure out how is coded.